src/base/bittorrent/addtorrentparams.h

```diff
@@ -51,12 +50,12 @@ namespace BitTorrent
         bool disableTempPath = false; // e.g. for imported torrents
         bool sequential = false;
         bool firstLastPiecePriority = false;
-        TriStateBool addForced;
-        TriStateBool addPaused;
+        bool addForced;
```

Comment:
Is this intended?

Reply:
>Is this intended?

Yes. We use optional paramters to override global settings. There is no global setting for it (and it is meaningless to have it).
But I missed to add initial value for it here. Will fix.

Reply:
Done.
PR rebased.